Formula/opencv.rb

```diff
@@ -143,11 +161,11 @@ def install
         return 0;
       }
     EOS
-    system ENV.cxx, "-std=c++11", "test.cpp", "-I#{include}/opencv4",
-                    "-o", "test"
-    assert_equal `./test`.strip, version.to_s
+    system ENV.cxx, "-std=c++11", "test.cpp", "-I#{include}/opencv4", "-o", "test"
+    assert_equal shell_output("./test").strip, version.to_s
 
-    output = shell_output(Formula["python@3.9"].opt_bin/"python3 -c 'import cv2; print(cv2.__version__)'")
+    python = Formula["python@3.10"].opt_bin/"python3.10"
+    output = shell_output("#{python} -c 'import cv2; print(cv2.__version__)'")
```

Comment:
Maybe can just use command:
```suggestion
    output = shell_output("python3.10 -c 'import cv2; print(cv2.__version__)'")
```